legend-engine-xt-mastery-pure/pom.xml

```diff
@@ -127,11 +127,6 @@
             <groupId>org.finos.legend.engine</groupId>
```

Comment:
This is not related to the relational refactoring .. right ? If yes, please move to a separate PR. This PR is big as is.

Reply:
Duplicate dependency declaration in this pom.xml. It was giving a warning during build, so removed it. Still want me remove it?